test-solution-template.groovy

```diff
@@ -30,6 +30,7 @@ def DeployJenkinsSolutionTemplate(scenario_name, options) {
     params['storageAccountType'] = ['value' : options.storageType]
     params['publicIPName'] = ['value' : 'jnktst' + UUID.randomUUID().toString().replaceAll('-', '')]
     params['dnsPrefix'] = ['value' : 'jnktst' + UUID.randomUUID().toString().replaceAll('-', '')]
+    params['jenkinsReleaseType'] = ['value' : options.jenkinsReleaseType]
```

Comment:
What is the point of this parameter if you're using the JOB_BASE_NAME to decide the release type? Suggest removing